script/tool/README.md

```diff
@@ -51,6 +51,8 @@ following shows a number of common commands being run for a specific plugin.
 All examples assume running from source; see above for running the
 published version instead.
 
+Some commands require the Flutter-bundled version of Dart to be the first `dart` loaded into path (or the only version of Dart installed).
```

Comment:
I would remove the parenthetical; having only one version of Dart install is just a subset of having that be the first version of `dart` in the path.

Also, let's lay "in the path" rather than "loaded into path".